docusaurus/sidebar-connectors.js

```diff
@@ -258,11 +258,11 @@ function getSourceConnectors(registry) {
       };
     });
   const sourcesWithSupportLevel = addSupportLevelToConnectors(
-    [...specialSources, ...sources, ...enterpriseSourcesWithSupportLevel],
```

Comment:
`enterpriseSourcesWithSupportLevel` already have `supportLevel` info, no need to check agains registry as we rely on the `enterprise` folder to categorize these.